src/main/build/debug.c

```diff
@@ -117,4 +117,5 @@ const char * const debugModeNames[DEBUG_COUNT] = {
     "MAG_CALIB",
     "MAG_TASK_RATE",
     "EZLANDING",
+    "S_TERM",
```

Comment:
Using `STERM` here would follow `ITERM` terminology.

> S-term stands for Setpoint (or Sticks or Servos term)

Make up your mind :)

Reply:
I think I_TRRM is easier to read, especially since this will be in the configuration drop-down. There is already D_MIN here, D_LPF. So probably ITERM_RELAX should be renamed in another PR :)